upup/pkg/fi/cloudup/azure/status.go

```diff
@@ -0,0 +1,226 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package azure
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2020-06-01/compute"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/klog"
```

Comment:
this should be `k8s.io/klog/v2` along with any other references to klog v1. the changes in go.mod shouldn't report adding `k8s.io/klog` after you've ran `make gomod`.